Change formatting of constitution. (#9)
UQComputingSociety/constitution: constitution.md

# Schedule Fourteen - The Constitution

## University of Queensland Computing Society

**Constitution - Affiliated Bodies**
**Version 1.4**
**30th of May 2019**

### 1 Preliminaries

> 1.1 The Society shall be called “University of Queensland Computing Society”, hereafter referred to as “the Society”  

Reviewer: ```suggestion
1.1 The Society shall be called “University of Queensland Computing Society”  
```

Author: Is this material enough to not be considered formatting?

Reviewer: Yes.

Author: Aight, I've removed it from this formatting PR.

> 1.1 The Society shall be called “University of Queensland Computing Society”, hereafter referred to as “the Society”  

Reviewer 2: Club/Society was changed to Society here, but not later in the document

Author: Depending on the resolution of the above comment I'll resolve this.

Author: See above.

* Educate our members through programming competitions, peer-workshops, group projects and other methods.
* Provide and facilitate educational, industry, corporate networking through various events, mixers, and connections.

2.2 The assets and income of the Club/Society shall be applied solely in furtherance of its above- mentioned objects and no portion shall be distributed directly or indirectly to the members of the Club/Society except as bona fide compensation for services rendered or expenses incurred on behalf of the Club/Society.
2.3 The Society shall not adopt aims or objectives that would bring it under the jurisdiction of the University of Queensland Sports Association (UQ Sport).

### 3 Interpretation

3.1 In this Constitution, unless the contrary intention appears:
    * “Semester” refers to Semester 1 or 2 as prescribed by the University's Academic Calendar

### 4 Membership

4.1 Membership is open to:

* any student of the University of Queensland
* any University of Queensland Academic Staff Member
* any other members of the University or wider community interested in Club/Society's activities

> 4.2 The Society shall have no less than thirty (30) members.

Reviewer: ```suggestion
4.2 The Society shall have no less than:
* Thirty (30) financial members in the case of a subsidised club
```

Author: Good call.

> 4.3 Fifteen (15) financial members in the case of a publications club

Reviewer: ```suggestion
* Fifteen (15) financial members in the case of a publications club
```

> 4.4 Ten (10) financial members in the case of an Ipswich based club.

Reviewer: ```suggestion
* Ten (10) financial members in the case of an Ipswich based club.
```

> 4.5 Seventy percent (70%) of the Society's financial members must be University of Queensland Union members.

Reviewer: ```suggestion
4.2 Seventy percent (70%) of the Society's financial members must be University of Queensland Union members.
```

2.2 The assets and income of the Club/Society shall be applied solely in furtherance of its above- mentioned objects and no portion shall be distributed directly or indirectly to the members of the Club/Society except as bona fide compensation for services rendered or expenses incurred on behalf of the Club/Society.

2.3 The Society shall not adopt aims or objectives that would bring it under the jurisdiction of the University of Queensland Sports Association (UQ Sport).

### 3 Interpretation

3.1 In this Constitution, unless the contrary intention appears:
    * “Semester” refers to Semester 1 or 2 as prescribed by the University's Academic Calendar

### 4 Membership

4.1 Membership is open to:

* any student of the University of Queensland
* any University of Queensland Academic Staff Member
* any other members of the University or wider community interested in Club/Society's activities

4.2 The Society shall have no less than:

* Thirty (30) financial members in the case of a subsidised club
* Fifteen (15) financial members in the case of a publications club
* Ten (10) financial members in the case of an Ipswich based club.

> 4.3 Seventy percent (70%) of the Society's financial members must be University of Queensland Union members.

Reviewer: ```suggestion
4.3 70% of the Club/Society's membership must be students.
```
